### ISSUE TITLE
Guide on how to document EA projects. 

### ISSUE THREAD

**Issue by ac-willeke** [NINAnor/EAguide#7]
@anders-kolstad, would be great if in the future you could add a short guide on how to best document your EA projects? 
Here you can maybe refer to the NINA rapport quarto template from @jenast and your template for creating a quarto webpage/book.

**Comment by anders-kolstad** [comment]
Yes, good idea. We can describe how to make a web site using ou rquarto book template. Also how to document indicator workflows using [ecRxiv](https://github.com/NINAnor/ecRxiv). I don't have capaciaty to start this work anytime soon though, but I put it on my to do list.
